sdk/timeseriesinsights/Azure.IoT.TimeSeriesInsights/samples/Readme.md

```diff
@@ -292,9 +350,34 @@ for (int i = 0; i < getTypeByIdResults.Value.Length; i++)
 }
 ```
 
-### Replace types
+Similarly, you can delete Time Series types by providing a list of Time Series type Ids or names.
```

Comment:
is this how Ids should be referred to? I really don't know and just wanted to probe anyone else giving feedback

* id's?
* ids?
* Id's?

Reply:
Good question. Also, I think it should be ID not Id. I believe we use Id in code, ID for documentation. 

Regarding the plural part, not sure. Let me do some digging.